### ISSUE TITLE
Museum Rework

### ISSUE THREAD

**Issue by KeremBabaG** [CleverRaven/Cataclysm-DDA#73544]
### Is your feature request related to a problem? Please describe.

Our museums are bad. I mean they are good locations to loot but they are kinda game-y and boring and do not reflect real life.

### Solution you would like.

Rework the current museum into new museums, with different themes and subjects which reflect the real life museums. Some ideas I came up ith are:

- Natural history museums. These are, well, museums of earth's natural history. Things like dinasour bones, models of extinct animals, history of evolution, displays of ecological diversity on earth can be found here.
- American history museums. This is the museum of the country of stars and stripes. From pre-colonization native nations to independence of 13 colonies, from westward expansion to civil war, from world wars to civil rights movement, this museum displays the US history as we know. The problem with this one however is how much the game's America deviates from our timeline. If some lore guy would give their ideas about the applicability of this one that'd be great.
- Civil war museum. Similar to above but just about Civil War. Loaded with muskets and bayonets and whatnot. It shares the same problem with the above tho.
- Art museums. It's about art and that's it. It can be branched further into its own epochs, like modern, post-modern, romanticism etc.
- Medieval hobbyist museum. Pretty much our current museum, minus the non medieval objects. Full of melee weapons and armor, most of which are fake but some can be real. I have a plan to make it more interesting but its still WIP.

And so on. I'll begin working on them if approved.

### Describe alternatives you have considered.

ehhhh just let it be

### Additional context

If you guys have any ideas you can write them below.

**Comment by harakka** [comment]
https://github.com/CleverRaven/Cataclysm-DDA/issues/64160

**Comment by KeremBabaG** [comment]
Damn

Should I close this one? That one's dead

**Comment by harakka** [comment]
No, just pointing you to previous ideas on the topic. You don't need a permission to work on these, museums that are more like real life museums would be very welcome.

**Comment by ADekema** [comment]
American history museum is interesting. We do have a few native american weapons and possibly some armours. Could be interesting to include them.

**Comment by PGR-14** [comment]
A couple of ideas:
-Raptor & Lizard samples in museums (T-Rex & others in dinomod)
-WW2 Museum (M2 Flamethrower, Garand, ect.)

**Comment by XygenSS** [comment]
None of the "equipment" you get in museums should be remotely useable right? The ones actually on display are old, rusted, rotted and barely preserved, and any replicas for show should be just as bad if not worse than the current fake weapons we have

**Comment by TheSaddestGoomba** [comment]
Would it be possible to have individual wings/halls generate independently? I ask because it could increase variability and create a place for future content to plug into.

A lot of museums have spaces for traveling exhibits that may not fit the usual theme or may even be empty at times. One example would be an exhibit of medieval European arms and armor at a museum that otherwise focuses on natural history.

I have plans for one such exhibit based on Body Worlds that would feature unique plastinated zombies.

**Comment by Maleclypse** [comment]
As harakka said more/better museums would be welcome content.

**Comment by Procyonae** [comment]
> Would it be possible to have individual wings/halls generate independently? I ask because it could increase variability and create a place for future content to plug into.

Yup that's possible with nests for smaller scale stuff and mutables for larger variations.
Parameters can also be used to make multiple different types of museum using the same layout for more variation with less work and maintenance (3 kinds of museum * 3 layouts = 9 potential themed museums for example)
